packages/asset-swapper/src/utils/market_operation_utils/sampler_operations.ts

```diff
@@ -838,7 +841,10 @@ export class SamplerOperations {
                     };
                 });
             },
-            () => [],
+            () => {
+                logUtils.warn('SamplerContractOperation: Two hop sampler reverted');
```

Comment:
Delete?

Reply:
I think keep so we can catch it when looking at logs, similar to the base sampler contract operation.

https://github.com/0xProject/protocol/blob/development/packages/asset-swapper/src/utils/market_operation_utils/sampler_contract_operation.ts#L59